hack/ensure-opm.sh

```diff
@@ -20,4 +19,4 @@ if [ ! -x "${OPM}" ]; then
 	chmod +x ${OPM}
 else
 	echo "Using opm cached at ${OPM}"
-fi
\ No newline at end of file
+fi
```

Comment:
Remove this change

Reply:
Why? all other bash scripts have a new line in the end
It is a common trait of many editors to add a newline at the end of a source file

Reply:
I removed it